runtime/oti/vm_api.h

```diff
@@ -485,7 +485,11 @@ popLoadingOrLinkingStack(J9VMThread *vmThread, J9StackElement **stack, J9Pool *s
 J9Class *
 internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass,
 	UDATA options, J9Class* elementClass, j9object_t protectionDomain, J9ROMMethod ** methodRemapArray,
-	IDATA entryIndex, I_32 locationType, J9Class *classBeingRedefined, J9Class *hostClass);
+	IDATA entryIndex, I_32 locationType, J9Class *classBeingRedefined, J9Class *hostClass
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	, J9Class *companionClass
```

Comment:
nitpick, the comma can be on `hostClass, `

Reply:
Won't compile in non-Valhalla build if comma moved on `hostClass, `